app/routes/password/reset.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+const { Route, inject: { service } } = Ember;
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+export default Route.extend(AuthenticatedRouteMixin, {
+
+  ajax: service(),
+
+  actions: {
+    resetPassword(password, passwordConfirmation) {
+      return this.get('ajax').request('/password/reset', {
+        method: 'POST',
+        data: {
+          password,
+          passwordConfirmation
```

Comment:
@JoshSmith so on the backend we do have a check `validate_confirmation` check 

https://github.com/code-corps/code-corps-api/blob/develop/web/models/user.ex#L101